truffle/src/main/ruby/core/exception.rb

```diff
@@ -189,12 +189,11 @@ def initialize(*args)
 end
 
 class NoMethodError < NameError
-  attr_reader :args
 
   def initialize(*arguments)
     super(arguments.shift)
     @name = arguments.shift
```

Comment:
Is this going to work?
Maybe at this point it would make sense to move `initialize` of exceptions in Java.